public/app/plugins/datasource/prometheus/querybuilder/components/promQail/state/helpers.ts

```diff
@@ -193,12 +302,25 @@ export async function promQailSuggest(
     if (interaction?.suggestionType === SuggestionType.AI) {
       feedTheAI = { ...feedTheAI, prompt: interaction.prompt };
 
-      // TODO: filter by metric type
       // @ts-ignore llms types issue
       results = await llms.vector.search<TemplateSearchResult>({
         query: interaction.prompt,
         collection: promQLTemplatesCollection,
         topK: 5,
+        filter: {
+          $or: [
+            {
+              metric_type: {
+                $eq: metricType,
```

Comment:
the `metric_type` filter currently only supports single metrics at a time, we should either change `histogram,summary` to return a single type, or change the filtering here so we add another OR clause

Reply:
oops yes sorry, I forgot about this special case